adb_graphics/figures/maps.py

```diff
@@ -322,24 +332,14 @@ def draw(self, show=False): # pylint: disable=too-many-locals, too-many-branches
 
         # Add wind barbs, if requested
         add_wind = self.field.vspec.get('wind', False)
-        if add_wind:
+        if add_wind and wind_barbs:
             self._wind_barbs(add_wind)
```

Comment:
Now we need the default_specs to have the `add_wind` and the argument to the `_draw_panel` method to also be true before we draw the winds. This allows us to turn off winds for all multipanel plots with one switch.